tools/run_disk_full_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TEST_MARK=${1:-disk_full_tests_5k}
+TMPFS_SIZE=${2:-5k}
+
+# Define a temporary mount point in /tmp
+MOUNT_POINT="/tmp/test_disk_full"
+
+# Create the namespace and mount tmpfs
+unshare --user --map-root-user --mount --fork --pid --mount-proc bash <<EOF
```

Comment:
Why do we need a namespace?

Reply:
to avoid running as root - I really try to avoid running things/tests as root when there is a solution

Reply:
Right, I forgot that `tmpfs`/`overlayfs` are allowed in namespaces.